x-pack/plugins/security_solution/public/flyout/shared/hooks/use_fetch_field_value_pair_with_aggregation.test.ts

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useQuery } from '@tanstack/react-query';
+import type { RenderHookResult } from '@testing-library/react-hooks';
+import { renderHook } from '@testing-library/react-hooks';
+import { useKibana } from '../../../common/lib/kibana';
+import type {
+  UseFetchFieldValuePairWithAggregationParams,
+  UseFetchFieldValuePairWithAggregationResult,
+} from './use_fetch_field_value_pair_with_aggregation';
+import { useFetchFieldValuePairWithAggregation } from './use_fetch_field_value_pair_with_aggregation';
+import { useDeepEqualSelector } from '../../../common/hooks/use_selector';
+import { useGlobalTime } from '../../../common/containers/use_global_time';
+
+jest.mock('@tanstack/react-query');
+jest.mock('../../../common/lib/kibana');
+jest.mock('../../../common/hooks/use_selector');
+jest.mock('../../../common/containers/use_global_time');
+
+const field = 'host.name';
+const values = ['values'];
+const isActiveTimelines = true;
+const aggregationField = 'aggregationField';
+
+describe('useFetchFieldValuePairWithAggregation', () => {
+  let hookResult: RenderHookResult<
+    UseFetchFieldValuePairWithAggregationParams,
+    UseFetchFieldValuePairWithAggregationResult
+  >;
+  (useKibana as jest.Mock).mockReturnValue({
+    services: {
+      data: { search: jest.fn() },
+    },
+  });
+  (useDeepEqualSelector as jest.Mock).mockReturnValue({ to: '', from: '' });
+  (useGlobalTime as jest.Mock).mockReturnValue({ to: '', from: '' });
```

Comment:
consider adding a test to check it is pulling the correct to and from based on isActiveTimelines

Reply:
yeah that's overkill for sure. My comment was about both of these 2 sets of `from` and `to` are empty strings, it would appears that whether `isTimelines` is true of not would yield the same results. Maybe change the return value to `timelineTo` or something, and check that when `isTimelines` is true, the correct params are being used, does that make sense?

Reply:
ok I finally got some time to look at this, and it's not straightforward. The `from` and `to` values are passed to the `buildSearchRequest` which isn't exported. Then the result of this is passed to `useQuery`.

Best way would be to export the line into its own hook but that definitely seems overkill to me. Other solution would be to export the `buildSearchRequest` method to be able to use it in the tests, but I'm not a fan of exporting things like that just for the sake of unit tests.

I'm tempted to not test this line, wdyt?